### PR TITLE
🐛 Source Monday: update schema for streams items and updates

### PR DESCRIPTION
**Pull request by arsenlosenko**
## What
Fix failed CAT run for Source Monday

## How
* Update schema for streams `items` and `updates`: field `id` in field `assets` sometimes could be a string
* Updated expected records for stream `items`, left only minimal amount of records
* Updated expected records for stream `updates`

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`cd07199601`](https://github.com/airbytehq/airbyte/pull/28076/commits/cd0719960133cdb6cefcfd947799172303ba6c37)) - ✅

⏲️  Total pipeline duration: 02mn41s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-monday-fix-expected-records-07-10/1688981944/cd0719960133cdb6cefcfd947799172303ba6c37/source-monday/0.2.7/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`cd07199601`](https://github.com/airbytehq/airbyte/pull/28076/commits/cd0719960133cdb6cefcfd947799172303ba6c37)) - ✅

⏲️  Total pipeline duration: 02mn35s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-monday-fix-expected-records-07-10/1688982182/cd0719960133cdb6cefcfd947799172303ba6c37/source-monday/0.2.7/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

File: airbyte-integrations/connectors/source-monday/source_monday/schemas/updates.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "type": ["null", "object"],
         "additionalProperties": true,
         "properties": {
-          "id": { "type": ["null", "integer"] }
+          "id": { "type": ["null", "integer", "string"] }
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-monday/source_monday/schemas/updates.json:11]: Can normalization be applied to multiple types of data?

**Review by lazebnyi — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-monday/source_monday/schemas/updates.json (hunk 1 above)
Comment (arsenlosenko) [reply airbyte-integrations/connectors/source-monday/source_monday/schemas/updates.json:11]: this particular change was enough to make CAT tests passing, to me it seemed that this was enough, but I'll check again if we should leave only "string" type here

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`65597bfac1`](https://github.com/airbytehq/airbyte/pull/28076/commits/65597bfac10bfb2f8d3202d5ddad63b909f05789)) - ✅

⏲️  Total pipeline duration: 02mn30s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-monday-fix-expected-records-07-10/1689066530/65597bfac10bfb2f8d3202d5ddad63b909f05789/source-monday/0.2.7/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by arsenlosenko** [comment]
fix is not relevant, closing

**Pull request closed without merging**